README.md

```diff
@@ -126,3 +126,26 @@ are:
   * `--wallclock-only` to suppress most details
   * `--xlimits` and `--inset-xlimits` to "zoom in" on parts of the x-axis
 
+
+## Diffing Krun results files
+
+Benchmarking is often performed in order to test whether a change in a given
+VM improves or worsens its performance. Unfortunately, the difference between
+benchmark performance before and after a change is rarely simple. Users will
```

Comment:
The sentence starting "Users" is a bit awkwardly phrased, and I don't think the tables or diffing are restricted to output from Krun. So maybe we should just call this section "Diffing results files" and remove mentions of Krun?

Reply:
At the moment we only diff Krun results files, so anyone with a CSV file needs to convert it first. "Normal" tables can be produced directly from the CSV file with `warmup_stats`.  (see #23)

Reply:
Perhaps I should also add some context here, this bit of the README is for people with Krun results files, and the text at the top of the file is for people with CSV files (probably not obvious from the GH diff), so the naming of the section is consistent with the other sections.

Reply:
OK, but that means that we need to think about putting this text elsewhere, or reorganising the file. We want people who might not be using Krun to diff their stats, so we don't want to suggest to them that they can't.

Reply:
Right, but that needs some code adding to `warmup_stats`, so it'll come in another PR that fixes #23

Reply:
Oh... Um, maybe we should fix #23 first then. Because I was just about to show this to people who are using CSV files ;)